subprojects/gdk_rust/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "gdk-rust"
+version = "0.1.0"
+authors = ["Riccardo Casatta <riccardo@casatta.it>", "Nadav Ivgi <nadav@shesek.info>"]
```

Comment:
I should probably be removed here, there's barely any code left in this repo from my initial sketch implementation :)

Reply:
well, you are also contributing to reviewing, I think it's ok to have your name here. However, if you prefer to be removed, please do a commit yourself, I don't want to be the guy removing authors :)